README.md

```diff
@@ -326,6 +327,19 @@ you can check the
 [vc-test-suite-implementations](https://github.com/w3c/vc-test-suite-implementations)
 library.
 
+### Allure Reporting
+It's also possible to generate local allure reports for analyzing and debuging results.
```

Comment:
Can you add a link to "allure".  I suspect there will be people (like me) that don't know what that is.

Reply:
added a link a short description for the allure reporting framework